### PR TITLE
Tooltip & Bubble Chart Updates

### PR DESCRIPTION
**Pull request by marioaks**
- Fixed Tooltip CSS according to feedback
 - Fixed Table CSS overriding Tooltip CSS
 - Added Bubble Scaling for mobile
 - Switched topojson package to topojson-client
 - Upgraded eslint

### CODE REVIEW

**Review by apothuku — COMMENTED** [review]
Approve table + map changes, would let Alex review scatter plot stuff

File: src/js/bail-trends.js [hunk 1]
```diff
@@ -247,15 +247,28 @@ const createCasesScatterPlot = () => {
     ]
   };
 
-  // 4px radius - 100;
-  // 30px radius - 10000;
-  // 120px radius - 40000;
+  const radius_desktop_min = 4, radius_desktop_10k = 35;
+  const radius_mobile_min = 4, radius_mobile_10k = 21;
```
Comment (alexwgraves) [inline src/js/bail-trends.js]: Camel case please!

File: src/js/bail-trends.js [hunk 2]
```diff
@@ -247,15 +247,28 @@ const createCasesScatterPlot = () => {
...
+
+  const maxRadiusValue = Math.max(...Object.values(BAIL_CASES_SCATTER_PLOT).map(county => county.r));
+  const minRadiusValue = 100;
+
+  const radius_desktop_max = ((radius_desktop_10k * (minRadiusValue - maxRadiusValue) + radius_desktop_min * (maxRadiusValue - 10000)) / (minRadiusValue - 10000));
+  const radius_mobile_max = ((radius_mobile_10k * (minRadiusValue - maxRadiusValue) + radius_mobile_min * (maxRadiusValue - 10000)) / (minRadiusValue - 10000));
```
Comment (alexwgraves) [inline src/js/bail-trends.js]: Looks like you've got an extra set of parens around these expressions

**Review by alexwgraves — COMMENTED** [review]
A couple style comments but other than that looks good!

**Review by apothuku — APPROVED** [review]

**Pull request merged**